packages/pwa/pwa-kit-config.json

```diff
@@ -1,7 +1,61 @@
 {
```

Comment:
I think the name we are going with for this file is going to be `pwa-kit.config.json` slight difference, but different none the less